ext/rejectu/rejectu.c

```diff
@@ -7,6 +7,8 @@
 static VALUE mRejectu = Qnil;
 static VALUE idEncoding, idTo_s;
 
+static char *defaultToken = "?";
```

Comment:
What I meant was define `defaultToken` as a `VALUE` here and then initialize it in `Init_rejectu` with `rb_str_new2`.